inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -378,46 +376,42 @@ discretized Gamma.
 
 ## Results
 
-```{r, eval = params$use_epiparameter}
-si_epiparameter <- epiparameter::epidist(
-  params$epiparameter_pathogen,
-  "serial_interval"
+```{r, eval = params$use_epiparameter_database}
+si_epidist <- epiparameter::epidist_db(
+  disease =  params$disease_name,
+  epi_dist =  "serial_interval",
+  single_epidist = TRUE,
+  subset = is_parameterised
 )
-si_params <- si_epiparameter$param
-si_dist <- si_epiparameter$dist
-si_mean <- si_params[[1]]
-si_sd <- si_params[[2]]
+
+si_params <- get_parameters(si_epidist)
+si_dist <- family(si_epidist)
+si_mean <- get_parameters(si_epidist)[1]
+si_sd <- get_parameters(si_epidist)[2]
 ```
 
-```{r, eval = !params$use_epiparameter}
+```{r, eval = !params$use_epiparameter_database}
 si_mean <- params$si_mean
 si_sd <- params$si_sd
 si_dist <- params$si_dist
 si_cv <- si_sd / si_mean
-si_params <- epitrix::gamma_mucv2shapescale(mu = si_mean, cv = si_cv)
+si_epidist <- epidist(
+  disease = params$disease_name,
+  epi_dist = "serial_interval",
+  prob_distribution = params$si_dist,
+  summary_stats = create_epidist_summary_stats(mean = params$si_mean,
+                                               sd = params$si_sd),
+  auto_calc_params = TRUE)
```

Comment:
`auto_calc_params` is set to `TRUE` by default and this is very unlikely to change in the future. Therefore you can remove `auto_calc_params = TRUE` and it will do the same thing. If having it there is serving another function (e.g. readability) then it can be left as is.

Reply:
Addressed in https://github.com/epiverse-trace/episoap/pull/100/commits/a3db3b35a04a6d8629ab7e0d9e1fed856caf03df